src/components/markdown/references/references.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { useContext } from 'react'
+import PropTypes, { array } from 'prop-types'
+
+import { BibReferencesContext } from '../../page-template'
+
+const References = ({ children, citationStyle }) => {
```

Comment:
The `children` prop can be removed since it's not used, and `citationStyle` too (see my comment about a single format below).

```suggestion
const References = () => {
```